### PR TITLE
ENG-490

### PR DESCRIPTION
**Pull request by jknight-liatrio**

### CODE REVIEW

**Comment by steeeveo** [comment]
:star: PR built and available in a preview environment **liatrio-sample-app-api-pr-9** [here](http://sample-app-api.jx-liatrio-sample-app-api-pr-9.lead-dev.liatr.io)

**Comment by jknight-liatrio** [comment]
ENG-490 tag

**Comment by steeeveo** [comment]
:star: PR built and available in a preview environment **liatrio-sample-app-api-pr-9** [here](http://sample-app-api.jx-liatrio-sample-app-api-pr-9.lead-dev.liatr.io)

File: charts/springboot-local/values.yaml [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
-  repository: local-sample-app-api
-  tag: latest
+  repository: sample-app-api
   pullPolicy: IfNotPresent
 
 nameOverride: ""
 fullnameOverride: ""
 
 service:
+  name: nginx
```
Comment (cplee) [inline charts/springboot-local/values.yaml]: why nginx?

File: charts/springboot-local/templates/deployment.yaml [hunk 2]
```diff
@@ -21,20 +21,20 @@ spec:
     spec:
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```
Comment (cplee) [inline charts/springboot-local/templates/deployment.yaml]: why aren't we using a tag anymore?

**Comment by steeeveo** [comment]
:star: PR built and available in a preview environment **liatrio-sample-app-api-pr-9** [here](http://sample-app-api.jx-liatrio-sample-app-api-pr-9.lead-dev.liatr.io)

**Comment by steeeveo** [comment]
:star: PR built and available in a preview environment **liatrio-sample-app-api-pr-9** [here](http://sample-app-api.jx-liatrio-sample-app-api-pr-9.lead-dev.liatr.io)

**Pull request merged**